front/app/containers/SignUpInPage/index.tsx

```diff
@@ -15,10 +18,11 @@ import GetLocale, { GetLocaleChildProps } from 'resources/GetLocale';
 import { isNilOrError, endsWith } from 'utils/helperUtils';
 
 // events
+// TODO: Remove
```

Comment:
To do = remove the comment?

Reply:
Remove the whole file in this case, actually 😂